tests/runtime/dwarf

```diff
@@ -205,3 +205,10 @@ PROG struct Foo { int b; } uprobe:./testprogs/uprobe_test:uprobeFunction2 { prin
 EXPECT foo1->b = 123
 REQUIRES_FEATURE dwarf
 BEFORE ./testprogs/uprobe_test
+
+NAME kprobe inlined function
+PROG config = { probe_inline = 1 } BEGIN { exit() }
+     kprobe:str_has_prefix { }
```

Comment:
is this kernel function known to be commonly inlined?

Reply:
Yes, I looked in the Kernel source code for a function marked `always inlined` to test on.
Moreover, `str_has_prefix` seems like a stable target that won't disappear from the Kernel any time soon 🤞